project_generator/tool.py

```diff
@@ -113,13 +113,12 @@ def get_value(self, tool, key):
     def get_supported(self):
         return self.TOOLS.keys()
 
-def export(exporter, data, tool, env_settings):
+def export(exporter, project_workspace, tool, env_settings):
```

Comment:
it's projects workspace (all projects), therefore env_settings are apart. What do you think we can do here?